sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -443,6 +443,32 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
     using SetPathPermissionsResult = SetPathAccessControlListResult;
 
+    /**
+     * @brief A path that has been soft deleted.
+     */
+    struct PathDeletedItem final
+    {
+      /**
+       * The name of the path.
+       */
+      std::string Name;
+
+      /**
+       * The deletion ID associated with the deleted path.
+       */
+      Azure::Nullable<std::string> DeletionId;
```

Comment:
When would this be null?

Reply:
ditto for below two fields.

Reply:
I think this won't be null in the ListDeletedPaths scenario. 
But the response from "ListBlobsByHierarchy" return these fields nullable, I'm worried that once a null value is returned, the sdk will throw an exception if these fields are not nullable.